analyses/create-subset-files/01-get_biospecimen_identifiers.R

```diff
@@ -181,6 +181,13 @@ num_nonmatched <- ceiling(0.1 * num_matched_participants)
 # set the seed
 set.seed(opt$seed)
 
+#### Samples we need to include to run tp53_nf1_score --------------------------
+
+tp53_dnaseq <- c("BS_3E5C1PN1", "BS_KHSYAB3J", "BS_ZR75EKKX")
```

Comment:
May wanna put a comment that explains how these came from the 00-enrich-positive-examples notebook.